samples/ribbon/ribbondemo.cpp

```diff
@@ -651,7 +651,7 @@ void MyFrame::OnPrimaryColourSelect(wxRibbonGalleryEvent& evt)
 {
     wxString name;
     wxColour colour = GetGalleryColour(evt.GetGallery(), evt.GetGalleryItem(), &name);
-    AddText(wxT("Colour \"") + name + wxT("\" selected as primary."));
+    AddText(wxString("Colour \"") + name + wxString("\" selected as primary."));
```

Comment:
Are these explicit `wxString()` really needed? AFAICS the code should be fine without them.

Reply:
Thanks, I fixed that.  I was being overly cautious, assuming that adding a pointer to a wxString wouldn't compile.